packages/did-comm/src/types/message-types.ts

```diff
@@ -16,6 +16,9 @@ export interface IDIDCommMessage {
   next?: string
   from_prior?: string
   body: any
+  options?: {
```

Comment:
This options object should be one of the arguments to `packDidcommMessage`, not part of `IDIDCommMessage`.

Please add it as an optional to `IPackDIDCommMessageArgs` instead

Reply:
ahh I see what you meant now, thank you for clarifying, let me correct :)